r/content/specify_data_types_and_schemas.Rmd

```diff
@@ -0,0 +1,205 @@
+# Defining Data Types
+
+As discussed in previous chapters, Arrow automatically handles the conversion of objects from native R data types to Arrow data types.  
+However, you might want to manually define data types, for example, to ensure interoperability with databases and data warehouse systems.
+
+## Specify data types when creating an Arrow table from an R object
+
+### Problem
```

Comment:
This `Problem`/`Solution` formatting is inconsistent with the other cookbook chapters.

Reply:
I was planning on rewriting all of the other recipes with this format shortly in a follow-up PR.  My thinking here is that I'd rather get this merged first as I have multiple PRs for multiple chapters open and don't want to end up with too many conflicts.  Of course, that doesn't stop me adding this formatting to those PRs if they haven't been merged yet, which I will take a look at doing.

Reply:
As long as there is an issue/JIRA for the format update (so we don't lose track of it) I think that's fine.

Reply:
I've opened #68 for this now.